src/docs/product/performance/transaction-summary.mdx

```diff
@@ -103,7 +103,13 @@ The "Spans" tab displays a list of suspect spans that correspond to where most o
 
 Choose from one of several [metrics](#metrics) to sort spans and identify different types of problems. Filter spans to only see the span operation type you're interested in.
 
-Clicking on a span will give you more details about the specific span [group](#grouping). You can see the performance of the span over time and see a list of transaction events that contain the span.
+Clicking on a span will take you to its summary page that should give you more details about this specific span [group](#grouping). On the page you can see the performance of the span over time and see a list of transaction events that contain the span.
```

Comment:
```suggestion
Clicking on a span will take you to its summary page that will give you more details about this specific span [group](#grouping). On the page you can see the performance of the span over time and see a list of transaction events that contain the span.
```

I feel like it's better to assume it will give more details than leave the user wondering whether it will or won't.